### PR TITLE
Cleaned up some code

### PR DESCRIPTION
**Pull request by namanix**
I cleaned up some code. There were allot of unneeded comma's. If you want you can ignore my version number changes.

### CODE REVIEW

File: Extensions/xneko.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 //* TITLE XNeko **//
-//* VERSION 1.2 REV A **//
+//* VERSION 1.2.1 REV A **//
```
Comment (Wolvan) [inline Extensions/xneko.js]: This is not a valid version string. You either have the format `X.Y REV Z` or `X.Y.Z`
REV A = X.Y.0

Comment (namanix) [inline Extensions/xneko.js]: So what do you want me to make of it? 1.2 REV B or just 1.2.1?

Comment (Wolvan) [inline Extensions/xneko.js]: `1.2.1`. We are switching all extensions to a [semver](http://semver.org/) compliant format

**Comment by izub** [comment]
Looks good, only some codestyle improvements.

**Comment by nightpool** [comment]
I don't know if removing terminal commas is a code-style change we want. I'll have to talk to the other devs. 

If we do end up doing it though, it'll probably be with an automated style tool, so I'm not sure if we want this PR or not. Again, we'll discuss this and get back to you. Thanks though!

(tl;dr Closing for now, but may re-open depending on discussion in our public Gitter)

**Comment by 0xazure** [comment]
Trailing commas are [perfectly valid in ES5+](http://ecma262-5.com/ELS5_HTML.htm#Section_11.1.5), so while we thank you for your interest in contributing to the project, this is not a change we are interested in including at the present time.

If you're interested, please follow #209 for a discussion about out code style and style enforcement.

**Comment by nightpool** [comment]
@0xazure trailing commas are valid in 3+ in arrays, also. That answer doesn't give enough rules to make me confident whether trailing commas in objects are or are not valid in 3+.

**Pull request closed without merging**